server/zanata-frontend/src/app/editor/index.html

```diff
@@ -15,7 +15,13 @@
   <meta name="author" content="Red Hat">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="cleartype" content="on">
-
+  <link rel="preconnect" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic" crossorigin />
+  <link rel="preload" as="font" crossorigin type="font/woff"
+        href="../../zanata-war/target/zanata/resources/assets/fonts/zanata-v6/zanata.woff"  />
+  <link rel="preload" as="font" crossorigin type="font/tff"
+        href="../../zanata-war/target/zanata/resources/assets/fonts/zanata-v6/zanata.tff"  />
+  <link rel="preload" as="font" crossorigin type="font/eot"
+        href="../../zanata-war/target/zanata/resources/assets/fonts/zanata-v6/zanata.eot"  />
```

Comment:
Would a single client be expected to use all three of these? I got the impression that they were fallback options.

Then again, the example on this page lists multiple types: https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content

Reply:
We need the multiple types because I noticed some pages use the woff, and some use the eot and some use the tff. I think that is why they covered all the fallbacks in the example also.